src/services/__tests__/cdcSupportService.test.ts

```diff
@@ -0,0 +1,118 @@
+import * as t from "io-ts";
+import { CdcSupportAPIClient } from "../../clients/cdc-support";
+import { mockedUser } from "../../__mocks__/user_mock";
+import CdcSupportService from "../cdcSupportService";
+import { CitizenStatus } from "../../../generated/io-cdc-support-func-api/CitizenStatus";
+
+const mockedInfo = {};
+
+const mockInfo = jest
+  .fn()
+  .mockImplementation(() => t.success({ status: 200, value: mockedInfo }));
+
+const mockedCitizenStatus = {
+  number_of_cards: 1,
+  expiration_date: new Date()
+} as CitizenStatus;
+
+const mockGetStatus = jest
+  .fn()
+  .mockImplementation(() =>
+    t.success({ status: 200, value: mockedCitizenStatus })
+  );
+
+const api = {
+  info: mockInfo,
+  getStatus: mockGetStatus
+} as ReturnType<CdcSupportAPIClient>;
+
+describe("CdcSupportService#status", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should make the correct api call", async () => {
+    const service = new CdcSupportService(api);
+
+    await service.status(mockedUser.fiscal_code);
+
+    expect(mockGetStatus).toHaveBeenCalledWith({
+      body: {
+        fiscal_code: mockedUser.fiscal_code
+      }
+    });
+  });
+
+  it("should handle a success response", async () => {
+    const service = new CdcSupportService(api);
+
+    const res = await service.status(mockedUser.fiscal_code);
+
+    expect(res).toMatchObject({
+      kind: "IResponseSuccessJson"
+    });
```

Comment:
we should check the payload too, to be sure that any transformation happens

Reply:
there's a decoder that fails if any transformation happens.

Reply:
anyway, i'll add it, better safe than sorry.

Reply:
added in https://github.com/pagopa/io-backend/pull/1250/commits/7bcb5b3b74dfebf4465c6fe7d6937216e391825d